src/chainstate/stacks/db/mod.rs

```diff
@@ -962,6 +962,7 @@ impl StacksChainState {
             if let Some(get_schedules) = boot_data.get_bulk_initial_lockups.take() {
                 info!("Initializing chain with lockups");
                 let mut lockups_per_block: HashMap<u64, Vec<Value>> = HashMap::new();
```

Comment:
Nitpick: can we just replace `lockups_per_block: HashMap<_, _> = HashMap::new()` with `BTreeMap` ?

`BTreeMap`s are sorted by key, so we wouldn't need a secondary structure here. It may be more efficient, though it's probably not important, because the expensive thing here is all the MARF edits, not this outer loop.

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2184/commits/7ea12d1cc4474e731294fdaa39a530db540f7522